lib/compiler_rt/memcpy.zig

```diff
@@ -8,21 +9,283 @@ comptime {
     }
 }
 
-pub fn memcpy(noalias dest: ?[*]u8, noalias src: ?[*]const u8, len: usize) callconv(.C) ?[*]u8 {
-    @setRuntimeSafety(false);
+const CopyType = if (std.simd.suggestVectorLength(u8)) |vec_size|
```

Comment:
You don't need to put "Type" in type names. It's already known to be a type, no need to also mention that it's a type in the type name.

A better name would be `Element`.